### PR TITLE
Migrate to GitHub Actions.

### LINKED ISSUE #187: Replace Travis-CI by GitHub Actions
Unfortunately, Travis-CI recently became unusable for open source projects. Anyone to create a GitHub Actions config for django-formtools?

### PR DESCRIPTION
**Pull request by jezdez**
Travis CI has a new pricing model which places limits on open source.

* https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing
* https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works

Many projects are moving to GitHub Actions instead, including Jazzband projects:

* https://github.com/orgs/jazzband/projects/1
* https://github.com/jazzband-roadies/help/issues/206

This is based on https://github.com/jazzband/contextlib2/pull/26.

Fixes #187.

TODO:

* [x] @jezdez to add `JAZZBAND_RELEASE_KEY` to the repo secrets.

### CODE REVIEW

**Comment by claudep** [comment]
Great! Did you leave travis config file on purpose?

**Comment by jezdez** [comment]
@claudep Yeah, not done yet :)

File: tox.ini [hunk 1]
```diff
@@ -21,3 +15,12 @@ deps =
     django-31: Django>=3.1,<3.2
     django-master: https://github.com/django/django/archive/master.tar.gz
     -r{toxinidir}/tests/requirements.txt
+
```
Comment (hramezani) [inline tox.ini:18]: An extra blank line should be removed

**Review by hramezani — APPROVED** [review]
LGTM! Thanks :heart:

**Pull request merged**